docs/reference/mapping/types/object.asciidoc

```diff
@@ -66,9 +64,8 @@ PUT my_index?include_type_name=true
 }
 --------------------------------------------------
 // CONSOLE
-<1> The mapping type is a type of object, and has a `properties` field.
```

Comment:
Same thought here as in the `properties` documentation.

Reply:
I changes this.